### DISCUSSION TITLE
Modified Gradle Wrapper

### DISCUSSION THREAD

**Discussion by stepango** [formatools/forma#57] [Ideas]
Gradle notation for referencing the projects is pretty unusual and creates some confusion:

```
:project
parent:sub:project - also valid 
:parent:sub:project
```

Bazel notation on the other hand is more straight forward and match with folder structure
```
//project
//parent/sub/project
```

Unfortunately, by default, there is no way to teach Gradle new project notations, but we can fork Gradle Wrapper so it can support Bazel notations and translate them to Gradle's.

**Comment by michaem** [comment]
For me, Bazel notation looks more logical, cause even on Gradle, :project notation, underhood, is directories too. 

For example, let to see on Gradle projects dir project(":feature-login").projectDir = file("feature/login"). 
Gradle project(...) tells to pass argument as `The Path`. It not same as directory, it just a value, which we use in the Gradle Build System and describes `The Project`, but underhood it always directory and prefer explicitly to set file(...) dir for project to avoid some bugs in showing project structure from IIDEA environment.

Thats why, will be great to choose unified notation for project/modules/target.

But on other hand, make this behavior on Gradle Wrapper side means, that we will forced to always support this forked version too)
